packages/mongodb-chatbot-eval/src/withConfig.test.ts

```diff
@@ -3,13 +3,16 @@ import { withConfig } from "./withConfig";
 
 describe("withConfig", () => {
   it("loads a config file", async () => {
+    const mockExit = jest.fn() as any;
+    process.exit = mockExit;
     await withConfig(
       async (config) => {
         expect(config.commands).toBeDefined();
         expect(config.evaluationStore).toBeDefined();
         expect(config.generatedDataStore).toBeDefined();
         expect(config.metadataStore).toBeDefined();
         expect(config.reportStore).toBeDefined();
+        expect(config.afterAll).toBeDefined();
```

Comment:
Do we actually want this to always be defined? We check if it's undefined in the actual implementation code + I can imagine a scenario where there's no cleanup to do (e.g. if you were using an Atlas Device SDK's remote MongoDB connection)

Reply:
not always, but this test just checks the specific config file.